app/models/adoption_application.rb

```diff
@@ -1,4 +1,32 @@
 class AdoptionApplication < ApplicationRecord
   has_many :application_pets
   has_many :pets, through: :application_pets
+
+  after_create :set_defaults
+
+  validates :name, presence: true
+  validates :street_address, presence: true
+  validates :city, presence: true
+  validates :state, presence: true
+  validates :zip_code, presence: true
+  validates :description, presence: true
+
+  def set_defaults
+    update(status: "In Progress")
+  end
+
+  def pending
+    update(status: "Pending")
+  end
+
+  def adopt(pet_id)
```

Comment:
also need this method explained. I don't typically use 'return if' and 'return unless' so I'm a bit confused.  Nice job though cause it clearly works!